tests/integration/security/sds_ingress/quic/ingress_test.go

```diff
@@ -78,6 +78,11 @@ func TestTlsGatewaysWithQUIC(t *testing.T) {
 		NewTest(t).
 		RequiresSingleCluster().
 		Run(func(t framework.TestContext) {
+			err := ingressutil.WaitForIngressQUICService(t, istio.DefaultSystemNamespace)
```

Comment:
nit: replace `istio.DefaultSystemNamespace` with `inst.Settings().SystemNamespace` here and below

Reply:
Done